next-cloudinary/src/components/CldImage/CldImage.js

```diff
@@ -56,6 +56,11 @@ const CldImage = props => {
     }
   }
 
+  if (props.src && props.useUrlTransformations) {
+    const transformations = getTransformations(props.src, props.useUrlTransformations);
+    imageProps.rawTransformations = transformations;
```

Comment:
i think we may want to concatenate the transformations onto `rawTransformations` in case someone is passing in additional transformations